OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -51,7 +51,24 @@ The parent class, Muscle.h, provides
 
 namespace OpenSim {
 
+//==============================================================================
+//                   Millard2012AccelerationMuscle Exceptions
+//==============================================================================
+class Millard2012AccelerationMuscleInvalidFlag : public Exception {
```

Comment:
For this PR the internal method and its implementation/code are not important.  What is important is that we not have the `###MuscleInvalidFlag` exceptions, which are less informative than the `MuscleCannotEquilibrate`, which should be thrown. If using enums prevents the invalid flag case from ever being reached (which I agree was a bad implementation to begin with) then that seems like a reasonable solution so that there is no need for that type or exception.  You could also fix/address the internal methods at a later data. Which ever you prefer is fine with me.